src/main/java/seedu/address/model/activity/Expense.java

```diff
@@ -25,6 +26,26 @@ public Expense(int personId, Amount amount, String description) {
         this.amount = amount;
         this.description = description;
         this.isDeleted = false;
+        involvedIds = null;
+    }
+
+    public Expense(int personId, Amount amount, String description, int ... ids) {
+        this(personId, amount, description);
+        this.involvedIds = ids;
+    }
```

Comment:
I think need null check for `int ... ids`? Unless we want to allow this constructor to be potentially misused and let in null values when rightfully should call the other constructor?

Or maybe its no big deal lol

Reply:
There an overloaded one with no vararg. There's no situation where the constructor can be misused by the user, but I will add it in so that it is not misused by the programmer.